shelter/racommand.go

```diff
@@ -19,27 +19,52 @@ var sgxraCommand = cli.Command{
 
 EXAMPLE:
        # shelter mrenclave`,
-	/*	Flags: []cli.Flag{
+	Flags: []cli.Flag{
+                cli.StringFlag{
+                        Name:  "ip",
+                        Usage: "tcp socket ip to connect inclavared",
+                },
 		cli.StringFlag{
-			Name:  "none",
-			Usage: "none",
+			Name:  "port",
+			Usage: "tcp socket port to connect inclavared",
 		},
-		cli.StringFlag{
-			Name:  "none",
-			Usage: "none",
-		},
-	},*/
+                cli.StringFlag{
+                        Name:  "log-level",
+                        Usage: "set the level of log output",
+                },
+                cli.StringFlag{
+                        Name:  "attester",
+                        Usage: "set the type of quote attester",
+                },
+                cli.StringFlag{
+                        Name:  "verifier",
+                        Usage: "set the type of quote verifier",
+                },
+                cli.StringFlag{
+                        Name:  "tls",
+                        Usage: "set the type of tls wrapper",
+                },
+                cli.StringFlag{
+                        Name:  "crypto",
+                        Usage: "set the type of crypto wrapper",
+                },
+	},
 
 	SkipArgReorder: true,
 
 	Action: func(cliContext *cli.Context) error {
 
-		var socketAddr string
-		socketAddr = cliContext.String("addr")
-		//connect to inclavared by TCP socket
-		//ret := remoteattestation.RemoteTlsSetupTCP(socketAddr, (unsafe.Pointer)(&remoteMrenclave[0]), (unsafe.Pointer)(&remoteMrsigner[0]))
-		//connect to ra-tls-server by unix socket
-		ret := remoteattestation.RemoteTlsSetupSock(socketAddr, (unsafe.Pointer)(&remoteMrenclave[0]), (unsafe.Pointer)(&remoteMrsigner[0]))
+		tcpIp := cliContext.String("ip")
+		tcpPort := cliContext.String("port")
+		logLevelInit := cliContext.String("log-level")
+		attester := cliContext.String("attester")
+		verifier := cliContext.String("verifier")
+		tls := cliContext.String("tls")
+		crypto := cliContext.String("crypto")
+		//attestation based on enclave-tls in tcp socket
+		ret := remoteattestation.EnclaveTlsSetupTcpSock(tcpIp, tcpPort, logLevelInit, attester, verifier, tls, crypto)
```

Comment:
Enclave TLS now supports mutual attesttion. Do you need to add a mutual attestation option?

Reply:
Per my understanding, the initial target of mutual attestation is for inclavared and skeleton, because they have evidence(report) to do mutual attestation for each other.  It is a good option, and we can integrate this option in this stage.